source/index.html

```diff
@@ -1,12 +1,169 @@
+            Каталог SLIM
+          </a>
+        </li>
+        <li class="offer-list__item offer__item--power">
+          <h2 class="offer-list__title">Набор массы</h2>
+          <span class="offer-list__description">Заработать авторитет среди дворовых котов и даже собак? Серия Cat Energy
+            Pro
+            поможет вашему коту нарастить необходимые мышцы!</span>
+          <a class="offer-list__description-button offer-list__description-button--arrow" href="#">
+            Каталог PRO
+          </a>
+        </li>
+      </ul>
+    </section>
+    <section class="work">
+      <h2 class="work__title">Как это работает</h2>
+      <ul class="work-list">
+        <li class="work-list__item">
+          <img class="work-list__image" src="./img/icons/leaf.svg" alt="Листочек в зеленом квадрате.">
+          <p class="work-list__text">Функциональное питание содержит только полезные питательные вещества.</p>
+        </li>
+        <li class="work-list__item">
+          <img class="work-list__image" src="./img/icons/powder.svg" alt="Порошок в зеленом квадрате.">
+          <p class="work-list__text">Выпускается в виде порошка, который нужно лишь залить кипятком и готово.</p>
+        </li>
+        <li class="work-list__item">
+          <img class="work-list__image" src="./img/icons/eat.svg" alt="Вилка и нож в зеленом квадрате.">
+          <p class="work-list__text">Замените один-два приема обычной еды на наше функциональное питание.</p>
+        </li>
+        <li class="work-list__item">
+          <img class="work-list__image" src="./img/icons/alarm.svg" alt="Будильник в зеленом квадрате.">
+          <p class="work-list__text">Уже через месяц наслаждайтесь изменениями к лучшему вашего питомца!</p>
+        </li>
+      </ul>
+    </section>
+    <section class="before-after">
+      <div class="before-after__half">
+        <h2 class="before-after__title">Живой пример</h2>
+        <span class="before-after__text">Борис сбросил 5 кг за 2 месяца, просто заменив свой обычный корм на Cat
+          Energy
+          Slim. Отличный результат без изнуряющих тренировок! При этом он не менял своих привычек и по-прежнему
+          спит
+          по 16 часов в день. </span>
+        <ul class="before-after-list">
+          <li class="before-after-list__item">
+            <span class="before-after-list__value">5 КГ</span>
+            <span class="before-after-list__title">снижение веса</span>
+          </li>
+          <li class="before-after-list__item">
+            <span class="before-after-list__value">60 ДНЕЙ</span>
+            <span class="before-after-list__title">затрачено времени</span>
+          </li>
+        </ul>
+        <div class="before-after__price">
+          <span class="before-after__price-item">Затраты на питание:</span>
+          <span class="before-after__price-value">15000 РУБ.</span>
+        </div>
+      </div>
+      <div class="before-after__half comparisons">
```

Comment:
```suggestion
      <div class="before-after__half comparison">
```

единственное число, потому что тут только 1 сравнение